corehq/apps/app_manager/static/app_manager/js/details/column.js

```diff
@@ -45,8 +45,45 @@ hqDefine("app_manager/js/details/column", function () {
         self.original.case_tile_field = ko.utils.unwrapObservable(self.original.case_tile_field) || "";
         self.case_tile_field = ko.observable(self.original.case_tile_field);
 
+        self.tileRowStart = ko.observable(self.original.grid_y || 1);
+        self.tileRowOptions = [""].concat(_.range(1, 4));
+        self.tileColumnStart = ko.observable(self.original.grid_x || 1);
+        self.tileColumnOptions = [""].concat(_.range(1, 13));
+        self.tileWidth = ko.observable(self.original.width || 3);
+        self.tileWidthOptions = ko.computed(function () {
+            return _.range(1, 14 - (self.tileColumnStart() || 1));
+        });
+        self.tileHeight = ko.observable(self.original.height || 1);
+        self.tileHeightOptions = ko.computed(function () {
+            return _.range(1, 5 - (self.tileRowStart() || 1));
+        });
+        self.tileRowEnd = ko.computed(function () {
+            return Number(self.tileRowStart()) + Number(self.tileHeight());
+        });
+        self.tileColumnEnd = ko.computed(function () {
+            return Number(self.tileColumnStart()) + Number(self.tileWidth());
+        });
+        self.showInTilePreview = ko.computed(function() {
+            return self.tileRowStart() && self.tileColumnStart() && self.tileWidth() && self.tileHeight();
+        });
+        self.tileContent = ko.observable();
+        self.setTileContent = function () {
+            let content = self.header.val();
+            if (content) {
+                content += ": ";
+            }
+            if (self.useXpathExpression) {
+                // TODO: this isn't displaying
+                content += gettext("Calculated Property") + " #" + self.$index();     // copied from calculatedColLabel
```

Comment:
@orangejenny I can find some `calculatedColLabel` but I don't see anything that looks like this code? Where is it from?

Reply:
No. I have not gotten that to work yet. I am wondering if I should just revert it to the header only for now.